esm_analysis/carbon.py

```diff
@@ -256,27 +256,85 @@ def _check_variables(ds):
     return sensitivity
 
 
+@check_xarray(0)
+def co2_flx_ocean_sensitivities(ds):
+    """Compute sensitivity of oceanic co2_flux to changes in driver variables.
+
+    .. math: co2_flx_ocean = co2trans * (co2atm - co2ocean) * (1 - ice)
+
+    Args:
+        ds (xr.Dataset): containing cmorized variables:
+                         * co2_flx_ocean [kg m-2 s-1]: surface air-sea CO2 flux
```

Comment:
Should be fgco2 (http://clipc-services.ceda.ac.uk/dreq/mipVars.html)

Reply:
yes. in our coupled model the CO2flux is calculated on the atmospheric grid. therefore to access the decomposition output files, I need to take those. `fgco2` is in our oceanic model output stream and `co2_flx_ocean` is the atmospheric models output. its the same fields but regridded. unfortunately we dont usually write out transfer velocity on the ocean side, therefore I did it this way. I suppose we could write the formulation independent of oceanic side or atmospheric side: allowing both names?